notebook/nbextensions.py

```diff
@@ -199,60 +173,184 @@ def install_nbextension(path, overwrite=False, symlink=False, user=False, prefix
+        Otherwise do a system-wide install (e.g. /usr/local/share/jupyter/nbextensions).
+    prefix : str [optional]
+        Specify install prefix, if it should differ from default (e.g. /usr/local).
+        Will install to ``<prefix>/share/jupyter/nbextensions``
+    nbextensions_dir : str [optional]
+        Specify absolute path of nbextensions directory explicitly.
+    logger : Jupyter logger [optional]
+        Logger instance to use
+    """
+    nbext = _get_nbextension_dir(user=user, sys_prefix=sys_prefix, prefix=prefix, nbextensions_dir=nbextensions_dir)
+    dest = cast_unicode_py2(dest)
+    full_dest = pjoin(nbext, dest)
+    if os.path.lexists(full_dest):
+        if logger:
+            logger.info("Removing: %s" % full_dest)
+        if os.path.isdir(full_dest) and not os.path.islink(full_dest):
+            shutil.rmtree(full_dest)
+        else:
+            os.remove(full_dest)
+    
+    # Look through all of the config sections making sure that the nbextension
+    # doesn't exist.
+    config_dir = os.path.join(_get_config_dir(user=user, sys_prefix=sys_prefix), 'nbconfig')
+    cm = BaseJSONConfigManager(config_dir=config_dir)
+    for section in ['common', 'notebook', 'tree', 'edit', 'terminal']:
+        cm.update(section, {"load_extensions": {require: None}})
+
+def uninstall_nbextension_python(package,
+                        user=False, sys_prefix=False, prefix=None, nbextensions_dir=None,
+                        logger=None):
+    """Uninstall an nbextension bundled in a Python package."""
+    m, nbexts = _get_nbextension_metadata(package)
+    base_path = os.path.split(m.__file__)[0]
+    for nbext in nbexts:
+        dest = nbext['dest']
+        require = nbext['require']
+        logger.info("{} {}".format(dest, require))
+        uninstall_nbextension(dest, require, user=user, sys_prefix=sys_prefix, 
+            prefix=prefix, nbextensions_dir=nbextensions_dir, logger=logger)
+    
+
+def enable_nbextension_python(package, user=False, sys_prefix=False):
+    """Enable an nbextension associated with a Python package."""
+    m, nbexts = _get_nbextension_metadata(package)
+    base_path = os.path.split(m.__file__)[0]
+    for nbext in nbexts:
+        config_dir = os.path.join(_get_config_dir(user=user, sys_prefix=sys_prefix), 'nbconfig')
+        cm = BaseJSONConfigManager(config_dir=config_dir)
+        cm.update(nbext['section'], {"load_extensions": {nbext['require']: True}})
+    
+
+def disable_nbextension_python(package, user=False, sys_prefix=False):
+    """Disable an nbextension associated with a Python package."""
+    m, nbexts = _get_nbextension_metadata(package)
+    base_path = os.path.split(m.__file__)[0]
+    for nbext in nbexts:
+        config_dir = os.path.join(_get_config_dir(user=user, sys_prefix=sys_prefix), 'nbconfig')
+        cm = BaseJSONConfigManager(config_dir=config_dir)
+        cm.update(nbext['section'], {"load_extensions": {nbext['require']: False}})
+
```

Comment:
Both these methods are identical except `False/True`, make a common private method and these 2  a 1 liner ?